Documentation/ApiOverview/TSFE/Index.rst

```diff
@@ -166,9 +166,12 @@ Get the language of the current page as integer:
 Access frontend user information
 --------------------------------
 
+..  versionchanged:: 13.0
+    The variable has been removed with TYPO3 v13.
+
 ..  code-block:: php
 
-    // !!! discouraged
+    // !!! Not available anymore since TYPO3 v13
     $feUser = $GLOBALS['TSFE']->fe_user;
```

Comment:
I would not leave a wrong example in here. I would remove the code-block and instead link to the migration section

Reply:
Personally, I would like to remove the whole part. It was discouraged in the past, and therefore it should not be used.

Reply:
Maybe keep a link to the migration section in v13 and remove in v14? I think use of this was once wide-spread so devs will have to migrate old projects?

Reply:
Done, added a link to the migration.